### ISSUE TITLE
bro has orphaned and misnamed libraries

### ISSUE THREAD

**Issue by gtkramer** [clearlinux/distribution#1591]
Orphaned libraries in the bro-plugins subpackage:
```
#     18456   1472 -rwxr-xr-x   1 root     root      1505368 Nov 19 05:46 /usr/lib/libcaf_io.so.0.16.2
#     18457    216 -rwxr-xr-x   1 root     root       219216 Nov 19 05:46 /usr/lib/libcaf_openssl.so.0.16.2
#     18455   1636 -rwxr-xr-x   1 root     root      1674600 Nov 19 05:46 /usr/lib/libcaf_core.so.0.16.2
#     18454   2936 -rwxr-xr-x   1 root     root      3004432 Nov 19 05:46 /usr/lib/libbroker.so..
```
Misnamed libraries in the bro-plugins subpackage:
```
/usr/lib/libbroker.so..
```

**Comment by phmccarty** [comment]
Hmm, the upstream project appears to have been renamed to [zeek](https://www.zeek.org/). We should probably look into updating/renaming it.

The misnamed library issue is likely resolved in the latest release of zeek (3.0.1), though it should not pose a problem in practice, because it currently has the correct soname set (`libbroker.so.0`). The CMakeLists.txt where the `SOVERSION` was being set referenced two undefined variables. Those are now set in the latest zeek.

Not sure if the mis-installed libraries are corrected in zeek 3.0.1, though with the current version of bro, the nonstandard `INSTALL_LIB_DIR` cmake variable must be used to set their installation prefix.

**Comment by phmccarty** [comment]
I replaced the `bro` package with `zeek` back [in October](https://github.com/clearlinux/clr-bundles/commit/81462c7b3d1194fd83276809e380b4cc74395e5d), and these issues are fixed with zeek. Closing.
